modules/virt-defining-watchdog-device-vm.adoc

```diff
@@ -23,26 +23,28 @@ apiVersion: kubevirt.io/v1
 kind: VirtualMachine
 metadata:
   labels:
-    kubevirt.io/vm: vm2-rhel84-watchdog
+    kubevirt.io/vm: <vm-label> <1>
   name: <vm-name>
 spec:
   runStrategy: Halted
   template:
     metadata:
       labels:
-        kubevirt.io/vm: vm2-rhel84-watchdog
+        kubevirt.io/vm: <vm-label> <1>
     spec:
       domain:
         devices:
           watchdog:
             name: <watchdog>
-            i6300esb:
-              action: "poweroff" <1>
+            <watchdog-device>: <2>
```

Comment:
Looking at these[ RHEL docs](https://docs.redhat.com/en/documentation/red_hat_enterprise_linux/6/html/virtualization_administration_guide/section-libvirt-dom-xml-watchdog) would it be better (work?) to have as follows?

```suggestion
            model: <watchdog-device><2>
```

Reply:
Got feedback through slack that this doesn't work with kubevirt.